code-challenges.js

```diff
@@ -20,9 +20,20 @@ const fruit2 = "banana"
 
 // --------------------1) Create the code that determines if a given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-// Describe your process:
+// Describe your process: Based on their being 3 different tempatures we will use a conditional statemenet to check if the tempatures are below, at, or above the boiling points. We will also use relational operators to check if the tempature is greater than, less than, or equal too the different boiling points. I'll also use string interpolation to inject the variable into the console log. Lastly, we will console log each individual (3 different tempatures) to ensure when we run terminal it is the correct answer. 
```

Comment:
Excellent work in this section Zack!!  Your notes for the process were well explained and your logic looks great.  Great job 👏